app/assets/stylesheets/homePage/home.less

```diff
@@ -2,10 +2,17 @@
 @import "../mixins.less";
 @import "homePageStyle.less";
 
+.vertical-scroll-disabled {
```

Comment:
This should not be in global LESS scope. Should most likely be in homePageStyle, mixins, or locally within the container.